inputs/prod_customize.json

```diff
@@ -0,0 +1,7 @@
+{
+    "disable_plugins": [
+    ],
+
+    "enable_plugins": [
+    ]
+}
```

Comment:
Ultimately we could mark the prod_customize.json file as %config(noreplace) in the rpm, but then `/usr/share/` would not be an appropriate place for it.